scripts/quests/windurst/Mihgos_Amigo.lua

```diff
@@ -0,0 +1,134 @@
+-----------------------------------
+-- Mihgo's Amigo
+-----------------------------------
+-- !addquest 2 25
+-- Nanaa Mihgo: !pos 62   -4    240    241
+-- Ardea      : !pos -198 -6   -69     235
+-- Varun      : !pos 7.8  -3.5 -10.064 241
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/quests')
+require('scripts/globals/settings')
+require('scripts/globals/status')
+require('scripts/globals/titles')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.WINDURST, xi.quest.id.windurst.MIHGO_S_AMIGO)
+
+local gilPerTrade = 200
+
+quest.reward =
+{
+    fame = 4, -- Baesd on 212 yag necklaces needed to reach level 4 tenshodo fame.
+    fameArea = xi.quest.fame_area.NORG,
+    title = xi.title.CAT_BURGLAR_GROUPIE,
+}
+
+quest.sections =
+{
+    {
+        -- QUEST AVAILABLE
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getFameLevel(xi.quest.fame_area.NORG) >= 1 and
```

Comment:
I think fame should be 1 by default. Or does it start at 0?

Reply:
You may be right on this one, but I'd rather have the check in place than not until I'm sure.  Easy to remove later.